src/ui-shell/header.component.ts

```diff
@@ -9,13 +26,17 @@ import { I18n } from "./../i18n/i18n.module";
 			role="banner"
 			[attr.aria-label]="brand + ' ' + name">
 			<a
+				*ngIf="!skipTo"
 				class="bx--skip-to-content"
 				[href]="skipTo"
 				tabindex="0">
 				{{ i18n.get("UI_SHELL.SKIP_TO") | async }}
 			</a>
 			<ng-content select="ibm-hamburger"></ng-content>
-			<a class="bx--header__name" href="#">
+			<a
+				class="bx--header__name"
+				href="#"
+				(click)="navigate($event)">
```

Comment:
Sorry for the intrusion here but wouldn't it be better to have `(selected)="navigate($event)"` to have the full hamburger functionality available for test?
I'm still getting my kicks around angular so i don't even know if that's the right MO.

Reply:
`click` is the standard HTML click event :+1: `selected` is just our convention for generic item selection events

Reply:
(And in many places we're shifting towards `value` + `valueChange` or `change` where that makes sense)

Reply:
Got it. Thanks.